src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -513,6 +514,14 @@ def _restart_qemu(self):
     """Restart QEMU."""
     logs.log_warn('Connection to fuzzing VM lost. Restarting.')
     stop_qemu()
+
+    # Do this after the stop, to make sure everything is flushed
+    if os.path.exists(QemuProcess.LOG_PATH):
```

Comment:
add an else and log_error if the log doesn't exist.

Reply:
Good catch, thanks